doc/templates/class.rst

```diff
@@ -13,10 +13,9 @@
 
 .. autoclass:: {{ objname }}
    :inherited-members:
-
-   {% block methods %}
-   .. automethod:: __init__
-   {% endblock %}
+   :special-members: __init__
+   :exclude-members: set_output
+   :member-order: alphabetical
```

Comment:
unrelated but it was bugging to see the methods not alphabetically sorted

Reply:
But the `:exclude-members:` is what removes it from the docs right?

Reply:
yup

Reply:
also no need for a special block to document init, it can just be done via special-members

Reply:
This is something you want to add in this PR?

Reply:
it's already done: the previous template made sure that the init of each class was documented, but this can just be done with the `:special-members: __init__` sphinx option